### ISSUE TITLE
Copy Paste Not Working on Windows

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#2709]
## Bug Description

Per Claudio on #1440:

> Testing in QA Env. - PR#2387
> 
> Tested on a PC so using ctrl-c/ctril-v but it isn't working (unable to test on a mac at the moment). Unable to cut-and-paste

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by spacedmonkey** [comment]
Can you provide more details on OS version, browser version etc.

**Comment by swissspidy** [comment]
^ @csossi can you answer that?

**Comment by merapi** [comment]
Fix pushed to https://github.com/google/web-stories-wp/pull/2387 (I tested it on Chrome/Safari/FF macOS+Windows, Claudio will test only on Windows I guess).

**Comment by csossi** [comment]
Still unable to cut-and-paste ctrl-c/ctrl-v on Windows. Tested in PR#2387

Tested in
Win10

Chrome 83.0.4103.116
Firefox 76.0.1

**Comment by merapi** [comment]
@csossi PR preview on QA is now fixed, PTAL.

**Comment by csossi** [comment]
Verified in QA - PR#2387 (Testing Instructions from #1440)

**Comment by o-fernandez** [comment]
Don't have a win device to test, but got verification by a few teammates.
